src/wwwroot/template-editor/adapterService.js

```diff
@@ -354,15 +356,28 @@
      */
     function getCampaign(id, useEditorAsTemplate) {
       traceAdapterCall(getCampaign, arguments);
-      var campaign = {
-        id: id,
-        type: 'campaign',
-        name: 'Campaign ' + id,
-        attributes: {},
-        innerHTML: '',
-        children: []
-      };
-      return $q.resolve({ data: campaign });
+      return $http.get(`${RELAY_CONFIG.baseUrl}/accounts/${loginSession.accountId}/template/${id}`, {
+        params: {
+          useAsTemplate: useEditorAsTemplate
```

Comment:
I think the we could ignore this parameter.

Reply:
👍